### PR TITLE
Feature/date

### PR DESCRIPTION
**Pull request by trox115**
- Create calendar
- add step 3

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,17 +3,18 @@ import React, { useState } from 'react';
 import './App.scss';
 import InitialScreen from './components/InitialScreen/InitialScreen';
 import AppContext from './Context/App.context';
+import Date from './pages/Date/Date';
 import Duration from './pages/Duration/Duration';
 
 function App() {
-  const [entered, setEntered] = useState(false);
```
Comment (LuisPedro01) [inline src/App.tsx:9]: why did you delete this?

**Review by LuisPedro01 — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (trox115) [reply src/App.tsx:9]: this is correct

**Review by LuisPedro01 — APPROVED** [review]
